src/script/ceph-backport.sh

```diff
@@ -129,13 +129,10 @@ redmine_endpoint="https://tracker.ceph.com"
 
 function usage () {
     echo "Usage:"
-    echo "   $0 [BACKPORT_TRACKER_ISSUE_NUMBER] [MILESTONE] [--prepare]"
+    echo "   $0 [BACKPORT_TRACKER_ISSUE_NUMBER] [--prepare]"
```

Comment:
You also need to update the instructions above not to include the milestone in the command line.

Reply:
good catch; done